timeout: print the name of the feed that times-out

In my dozens of feeds, one times-out:

    [ERROR] 60 second timeout exceeded

It would be great to know which one to remove or update the feed URL.

--- a/rss2email/command.py
+++ b/rss2email/command.py
@@ -61,6 +61,8 @@ def run(feeds, args):
     try:
         for index in args.index:
             feed = feeds.index(index)
+            # to debug feeds that timeout, run "r2e -V run"
+            _LOG.warning('refreshing feed {}'.format(feed))
             if feed.active:
                 try:
                     feed.run(send=args.send)

--- a/rss2email/command.py
+++ b/rss2email/command.py
@@ -61,8 +61,8 @@ def run(feeds, args):
     try:
         for index in args.index:
             feed = feeds.index(index)
-            # to debug feeds that timeout, run "r2e -V run"
-            _LOG.warning('refreshing feed {}'.format(feed))
+            # to debug feeds that timeout, run "r2e -VV run"
+            _LOG.info('refreshing feed {}'.format(feed))
             if feed.active:
                 try:
                     feed.run(send=args.send)

--- a/rss2email/config.py
+++ b/rss2email/config.py
@@ -213,5 +213,5 @@ CONFIG['DEFAULT'] = _collections.OrderedDict((
 
         ### Miscellaneous
         # Verbosity (one of 'error', 'warning', 'info', or 'debug').
-        ('verbose', 'warning'),
+        ('verbose', 'info'),
         ))

